packages/cli/src/commands/build/index.ts

```diff
@@ -152,7 +152,11 @@ export default async function main(client: Client): Promise<number> {
   }
 
   // Build `target` influences which environment variables will be used
-  const target = parsedArgs.flags['--prod'] ? 'production' : 'preview';
+  const target = parsedArgs.flags['--prod']
```

Comment:
**non-blocking:** do we want to throw if both `--prod` and `--target` are provided?